src/diy/smol-slimes/smol-slimes-community-builds.md

```diff
@@ -11,7 +11,7 @@ This page is dedicated only to builds that others can build themselves. Builds t
 **Want to contribute your design?** Awesome! To have your build added to this list, please ensure the following:
 
 - Your case must be **publicly accessible** (e.g. GitHub(Preferable), Thingiverse, etc).
-- Include at least one clear image or render of your build with 2x1, 3x1 aspect ratio.
+- Include image or render of your build with matching aspect ratio of other images.
```

Comment:
This change is a bit vague, what is the aspect ratio for other images? It would be hard to check for most people.